net/port-mirroring/Makefile

```diff
@@ -1,18 +1,18 @@
 #
-# Copyright (c) 2015 OpenWrt.org
+# Copyright (c) 2019 OpenWrt.org
```

Comment:
Don't update this, you can't assign copyright to OpenWrt unless you're affiliated
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain